index.d.ts

```diff
@@ -1,5 +1,12 @@
 export type Property<T> = string | ((element: T) => unknown) | Array<string | ((element: T) => unknown)>;
 
+interface Options {
+	/** A locale string or array of locale strings that contain one or more language or locale tags. If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale. If you omit this parameter, the default locale of the JavaScript runtime is used. This parameter must conform to BCP 47 standards; see the Intl.Collator object for details. */
+	locales?: string | string[];
+	/** An object that contains one or more properties that specify comparison options. see the Intl.Collator object for details. */
+	localeOptions?: Intl.CollatorOptions;
```

Comment:
Do we have to document this? Can't it just inherit the docs from `Intl.CollatorOptions`?

Reply:
I copied the documentation from the parameter on `localeCompare`, not from the `Intl.CollatorOptions` interface. The later doesn't have a doc string attached to it.

I tried to type it as `Parameters<String['localeCompare']>[2]`, but that doesn't carry over the doc string either.

Open to suggestions 👍

Reply:
Maybe https://tsdoc.org/pages/tags/inheritdoc/ ?

Reply:
Unfortunately, I haven't manage to accomplish this with `@inheritDoc`. I don't see anything in the documentation on it to see that it could be used to lift the documentation from an `@param` and use it as the summary.

And on top of that, I cannot even get VS Code to show a copied summary, only the params description. I think that the support for this is quite weak...